siac/main.go

```diff
@@ -262,6 +262,9 @@ func main() {
 		renterContractsCmd, renterFilesListCmd, renterFilesRenameCmd,
 		renterFilesUploadCmd, renterUploadsCmd, renterExportCmd,
 		renterPricesCmd)
+
+	renterContractsCmd.AddCommand(renterContractDetailsCmd)
```

Comment:
I believe same thing here, the command name should be derived directly from the route name. Makes navigating 100 api calls a lot easier. Use comments to indicate what the command does if you don't think the name is nearly expressive enough.